src/app/flex-utility.scss

```diff
@@ -0,0 +1,76 @@
+.flex-row {
```

Comment:
The old file fx-utility.scss can be deleted

Reply:
it is deleted